packages/moonstone/Panels/Panel.js

```diff
@@ -173,7 +173,7 @@ const PanelBase = kind({
 		delete rest.hideChildren;
 
 		return (
-			<article role="region" {...rest} aria-labelledby={headerId} ref={spotOnRender}>
+			<article role="region" {...rest} aria-labelledby={headerId} ref={spotOnRender} style={{'--spatial-navigation-contain': 'contain', '--spatial-navigation-action': 'focus'}} >
```

Comment:
I prefer to use a constant in this case, but it looks okay to see what is needed for now.

Reply:
OK, I'll fix it in the next PR

Reply:
👍